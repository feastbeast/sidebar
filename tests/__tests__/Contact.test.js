import React from 'react';
import Enzyme from 'enzyme';
import {shallow, mount, render} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import Contact from '../../client/src/components/Contact';

Enzyme.configure({ adapter: new Adapter() });



describe('Contact', () => {
  

  it('should be defined', () => {
    let address= '944 Market St'; 
    let phone='888-8888';
    let website='google.com';
    let location='SF';
    let id='1234';
    let name='Hack Reactor';

    let wrapper = shallow(<Contact
      address={address} 
      phone={phone}
      website={website}
      location={location}
      id={id}
      name={name}
    />) 
    expect(wrapper).toBeDefined();
  });
  it('should render correctly', () => {
    let address= '944 Market St'; 
    let phone='888-8888';
    let website='google.com';
    let location='SF';
    let id='1234';
    let name='Hack Reactor';

    const tree = shallow(
      <Contact 
      address={address} 
      phone={phone}
      website={website}
      location={location}
      id={id}
      name={name}
      />
    );
    expect(tree).toMatchSnapshot();
  });
});