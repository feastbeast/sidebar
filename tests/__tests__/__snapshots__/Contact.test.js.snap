// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact should render correctly 1`] = `
<div
  className="contact"
>
  <Modal
    actions={
      Array [
        <Button
          flat={true}
          modal="close"
          node="button"
          waves="light"
        >
          Close
        </Button>,
      ]
    }
    bottomSheet={false}
    className="mapModal"
    fixedFooter={false}
    modalOptions={Object {}}
    trigger={
      <div
        className="address row"
      >
        <Icon
          className="contactIcon icons"
        >
          location_on
        </Icon>
        <span
          className="contactText"
        >
          944 Market St
        </span>
      </div>
    }
  >
    <iframe
      allowFullScreen={true}
      frameBorder="0"
      height="500px"
      src="https://www.google.com/maps/embed/v1/place?key=AIzaSyDfVc7Wl0xQOdHnSm30Yk2lZtcBTPEZtjM&q=place_id:1234"
      style={
        Object {
          "border": 0,
        }
      }
      width="1000px"
    />
  </Modal>
  <div
    className="phone row"
  >
    <a
      href="tel:888-8888"
      target="blank"
    >
      <Icon
        className="contactIcon icons"
      >
        phone
      </Icon>
      <span
        className="contactText"
      >
        888-8888
      </span>
    </a>
  </div>
  <div
    className="website row"
  >
    <a
      href="google.com"
      target="blank"
    >
      <Icon
        className="contactIcon icons"
      >
        web
      </Icon>
      <span
        className="contactText"
      >
        google.com
      </span>
    </a>
  </div>
  <div
    className="directions row"
  >
    <a
      href="https://www.google.com/maps/dir/?api=1&destination=Hack+Reactor&destination_place_id=1234"
      target="blank"
    >
      <Icon
        className="contactIcon icons"
      >
        directions
      </Icon>
      <span
        className="contactText"
      >
        Get Directions
      </span>
    </a>
  </div>
</div>
`;
